src/sagemaker/tensorflow/deploying_python.rst

```diff
@@ -0,0 +1,190 @@
+Deploying to Python-based Endpoints
+===================================
+
+Deploying from an Estimator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+After a ``TensorFlow`` Estimator has been fit, it saves a TensorFlow ``SavedModel`` in
+the S3 location defined by ``output_path``. You can call ``deploy`` on a ``TensorFlow``
+estimator to create a SageMaker Endpoint.
+
+A common usage of the ``deploy`` method, after the ``TensorFlow`` estimator has been fit look
```

Comment:
don't need "TensorFlow" to be monospace on ll. 8 and 11

Reply:
ok